Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp

```diff
@@ -167,7 +167,7 @@ bool RenderSVGResourceClipper::applyClippingToContext(GraphicsContext& context,
     if (!clipperData.imageBuffer && pathOnlyClipping(context, animatedLocalTransform, objectBoundingBox, effectiveZoom))
         return true;
 
-    if (clipperData.invalidate(computeInputs(renderer, objectBoundingBox, clippedContentBounds, effectiveZoom))) {
+    if (clipperData.invalidate(computeInputs(renderer, context, objectBoundingBox, clippedContentBounds, effectiveZoom))) {
```

Comment:
Should not `computeInputs() `and `applyClippingToContext()` have the arguments: RenderElement and GraphicsContext in the same order: `(const RenderElement&, const GraphicsContext& ...)` or `(const GraphicsContext&,  const RenderElement& ...)`?